src/autopas/utils/ArrayMath.h

```diff
@@ -214,6 +216,18 @@ template <class T, std::size_t SIZE>
   return dotAux(a, b, std::make_index_sequence<SIZE>{});
 }
 
+/**
+ * Generates the cross product of two arrays of 3 floats.
+ * @tparam T floating point type
+ * @param a 3D vector (denoted by array of 3 floats)
+ * @param b 3D vector (denoted by array of 3 floats)
+ * @return cross product a x b
+ */
+template <class T>
+[[nodiscard]] constexpr std::array<T, 3> cross(const std::array<T, 3> &a, const std::array<T, 3> &b) {
```

Comment:
I don't like how this is the only method here that is restricted to arrays of size 3. Can you please make this more generic?

Reply:
Ok then let's just leave it as it is.